lib/caching-precompiler.js

```diff
@@ -74,14 +65,16 @@ CachingPrecompiler.prototype._transform = function (code, filePath, hash) {
 };
 
 CachingPrecompiler.prototype._createTransform = function () {
-	var salt = packageHash.sync(
-		[require.resolve('../package.json')].concat(babelConfigHelper.pluginPackages),
-		JSON.stringify(this.babelConfig),
-		process.version.split('.')[0]
-	);
+	var pluginPackages = babelConfigHelper.pluginPackages;
+	var avaPackage = require.resolve('../package.json');
+
+	var packages = [avaPackage].concat(pluginPackages);
+	var packageSalt = JSON.stringify(this.babelConfig);
```

Comment:
You need to add `process.version.split('.')[0]` to this. Note that it's currently broken: https://github.com/avajs/ava/pull/1068/files#r83534244.